docs/ispc.rst

```diff
@@ -568,6 +569,29 @@ aligned, please refer to `Function Templates`_ section for more details.
 Now, command-line switch `--dwarf-version=<n>` forces DWARF format debug info generation
 on Windows. It allows to debug ISPC code linked with MinGW generated code.
 
+Updating ISPC Programs For Changes In ISPC 1.23.0
+-------------------------------------------------
+
+This release contains the following changes that may affect compatibility with
+older versions:
+
+* `true` `bool` values in storage were changed from `-1` to `1` to match ABI.
+  It may have resulted to incomparable `true` values with C/C++ `true` values
+  since 1.13.0 release.
+
+A couple of improvements have been made to variables initialization:
+
+* Variables with const qualifiers can be initialized using the values of
+  previously initialized const variables including arithmetic operations above
+  them. It now works also with varying types.
+* Enumeration type values can be used as constants.
+
+The result of selection operator can now be used as lvalue if it has suitable
+type.
+
+Now, command-line switch `--dump-file=<dir>` forces dumping of the whole IR
+modules after each pass.
```

Comment:
I don't think this option is needed to be described here since `--dump-file` is an option for ISPC developers (under `--help-dev`).